code/drasil-lang/lib/Language/Drasil/Document.hs

```diff
@@ -133,11 +133,11 @@ getSecCons (Sub sec) = getSec sec
 getSecCons (Con _)   = []
 
 -- | 'Figure' smart constructor with a 'Lbl' and a 'Filepath'. Assumes 100% of page width as max width.
-fig :: Lbl -> Filepath -> RawContent
+fig :: Lbl -> Filepath -> HasCaption -> RawContent
```

Comment:
Sorry to cause yet more rework, but I think the 'right' thing to do would be to have `fig`, the smart constructor, default to `WithCaption` and then have a `figNoCap` constructor for the other case.